web/src/main/ui/app/components/flows-new/models/flow.model.ts

```diff
@@ -27,7 +27,11 @@ export class Flow {
   }
 
   get status(): string {
-    return (this.latestJob && this.latestJob['status']) ? this.latestJob['status'] : 'Never Run';
+    if (this.latestJob === null) {
+      return '';
+    } else {
+      return (this.latestJob && this.latestJob['status']) ? this.latestJob['status'] : 'Never Run';
```

Comment:
Status text should be "Never run" (capitalize first word, lowercase those that follow).